app/models/chargeback_rate.rb

```diff
@@ -70,4 +72,9 @@ def self.seed
       end
     end
   end
+
```

Comment:
I know callbacks are changing in Rails 5, so this may not be the best way to do this. But if it works, it would help:
* clean up the controller, beceause it doesn't have to care about what ids are sent to it
* keep the behavior consistent between the UI and the API

If this doesn't/can't work for reasons I perhaps don't understand, let me know and I may have to update the API code to enforce this behavior there too.

Also apologies as my indentation is a bit off in the snippet above